content/onboarding.rst

```diff
@@ -145,30 +150,34 @@ Use the dedicated ProAtria Skype account as the official tools to collaborate wi
 * Request for VPN certificates to connect to internal servers/services.
 
 * You will need a Trac account setup.
-Trac tickets are used for managing work items since there is no support with the web-based GitHub issue/task/defect management
-Trac also contains wiki pages to other documentation.
-Get to know the team by checking the dedicated page in our private wiki.
+  Trac tickets are used for managing work items since there is no support with the web-based GitHub issue/task/defect management
+  Trac also contains wiki pages to other documentation.
 
+* Get to know the team by checking the dedicated page in our private wiki.
 
-4. Exploring SFTPPlus for the first time
-----------------------------------------
 
-When testing out the software, please take a look at the contents of these folders as it may contain useful files for exploring various features of SFTPPlus.  
+4. Using the product for the first time
```

Comment:
I think that this should be removed as it is too specific.

We can ove it into an internal wiki